dao/src/main/java/greencity/entity/notifications/NotificationTemplate.java

```diff
@@ -15,6 +12,7 @@
 @AllArgsConstructor
 @EqualsAndHashCode
 @NoArgsConstructor
+@Builder
```

Comment:
why do you need both @ Data and @ Builder implementations?  The both provide the way to build your object: first - using chain of setters and second - with builder methods.